README.md

```diff
@@ -101,6 +101,29 @@ await page.RouteAsync("**", (route, _) =>
 await page.GotoAsync("http://todomvc.com");
 ```
 
+## CLI & Codegen
+
+Playwright conveniently comes with a built-in [`codegen` facility](https://playwright.dev/dotnet/docs/cli) which lets you open an inspector and a browser window, then record the 
+actions you perform, generating the relevant source code as you do.
+
+To use it, you need to install the dotnet tool:
+
+```powershell
+dotnet-install --global microsoft.playwright.cli
+```
+
+Then, you can simply call:
+
+```powershell
+playwrightcli codegen https://www.microsoft.com
+```
+
+### Driver Discoverability
+
+By default, the tool will attempt to look at `%USERPROFILE%\.nuget\packages\microsoft.playwright` for the location of the driver. If that behaviour
```

Comment:
Will the tool pick up Playwright from my project, if I am at `src/`?

Reply:
I've changed the behaviour and doc now:

- traverse current folder and find `.playwright` 
- NuGet cache